### ISSUE TITLE
Entity relationship diagrams (erd, database, schema)

### ISSUE THREAD

**Issue by mimecuvalo** [tldraw/tldraw#4276]
### Discussed in https://github.com/tldraw/tldraw/discussions/3021

<div type='discussions-op-text'>

<sup>Originally posted by **jmatsushita** March  2, 2024</sup>
Hi there,

I saw [this tweet](https://x.com/tldraw/status/1734948064782061692?s=20) and thought that tldraw could produce entity relationship diagrams, but I think these are images that must have been generated by another software?

![image](https://github.com/tldraw/tldraw/assets/356097/16d96e3a-76ba-4caa-a2b6-d4c91cfc29d8)

What would it take to allow tldraw to draw entity relationship diagrams?

Cheers,

Jun
</div>

**Comment by mrn3** [comment]
Is there any progress on this?  I don't have access to that Linear link.

**Comment by steveruizok** [comment]
Yes, those images come from somewhere else (I think I googled "entity relationship diagram"). Draw.io has some good primitives for these types of chart. We don't really produce diagrams at the moment on tldraw.com, though you could use teach.tldraw.com to generate things!
